### DISCUSSION TITLE
Chercher une cantine avec le SIRET sur "admin"

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1564] [Idées]
**Problème :**
Il m'est arrivé plusieurs fois de chercher sur notre back office une cantine avec son nom, mais de ne pas le trouver car le nom de la cantine n'avait pas été bien renseigné (ex. : "restaurant scolaire" uniquement renseigné).

**Solution envisagée :**
Avoir un champ de recherche "SIRET" sur la page "Cantine" de notre back office.

@JenniferStephan, @hfroot, @alemangui, votre avis?

**Comment by hfroot** [comment]
C'est maintenant possible sur staging (pour les cantines, diagnostics et invitations de gestion)

**Reply by ghost** [reply]
C'est une amélioration qui me sera bien utile. Merci, @hfroot!
